lib/chooseTeam.js

```diff
@@ -0,0 +1,33 @@
+const rl = require("./rl");
+
+const promptForTeamChoice = () => {
+  return new Promise((resolve) => {
+    rl.question("Pick your team. O or X? = ", (choice) => {
+      return resolve(choice.toUpperCase());
+    });
+  });
+};
+
+const isValidTeamChoice = (choice) => {
+  if (choice === "O" || choice === "X") {
+    return true;
+  } else {
+    console.log("Please enter only the letter O or X to choose your team");
+    return false;
+  }
+};
+
+const getTeamChoice = async () => {
+  let x = false;
+  while (x === false) {
```

Comment:
it's impossible for x to be true here because of your return statement.

In most places, it's cleaner to use `while (true)` and use `return` or `break` to escape from the loop.

```suggestion
  while (true) {
```